config/locales/strings/en.posts.yml

```diff
@@ -64,3 +64,5 @@ en:
       In a URL of "https://yoursite.codidact.com/help/topic", the "topic" is the slug.
     responding_to: >
       Responding to:
+    no_block_mathjax_title: >
+     Title cannot contain block-level MathJax.
```

Comment:
You need one more space at the start of this line. YAML is very picky about indentation.